specification/devcenter/data-plane/Microsoft.DevCenter/preview/2023-07-01-preview/devbox.json

```diff
@@ -1978,6 +2038,196 @@
+          "type": "string"
+        },
+        "createdByObjectId": {
+          "description": "The object ID of the actor which initiated the operation.",
+          "type": "string"
+        },
+        "startTime": {
+          "description": "The time the operation started.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "endTime": {
+          "description": "The time the operation finished.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "error": {
+          "description": "Provisioning or operation error details. Populated only for error states.",
+          "$ref": "devcenter.json#/definitions/CloudErrorBody"
+        }
+      },
+      "required": [
+        "uri",
+        "operationId",
+        "kind",
+        "status"
+      ]
+    },
+    "DevBoxStartOperation": {
+      "type": "object",
+      "description": "Information about a start operation on a Dev Box.",
+      "x-ms-discriminator-value": "Start",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DevBoxOperation"
+        }
+      ]
+    },
+    "DevBoxStopOperation": {
+      "type": "object",
+      "description": "Information about a stop operation on a Dev Box.",
+      "x-ms-discriminator-value": "Stop",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DevBoxOperation"
+        }
+      ]
+    },
+    "DevBoxRestartOperation": {
+      "type": "object",
+      "description": "Information about a restart operation on a Dev Box.",
+      "x-ms-discriminator-value": "Restart",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DevBoxOperation"
+        }
+      ]
+    },
+    "DevBoxRepairOperation": {
+      "type": "object",
+      "description": "Information about a repair operation on a Dev Box.",
+      "x-ms-discriminator-value": "Repair",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DevBoxOperation"
+        }
+      ],
+      "properties": {
+        "result": {
+          "description": "The result of the repair operation.",
+          "$ref": "#/definitions/DevBoxRepairOperationResult"
+        }
+      }
+    },
+    "DevBoxRepairOperationResult": {
+      "description": "Information about the result of a repair operation on a Dev Box.",
+      "type": "object",
+      "properties": {
+        "repairOutcome": {
+          "description": "The outcome of the repair operation.",
+          "$ref": "#/definitions/DevBoxRepairOutcome"
+        },
+        "code": {
+          "description": "The result code associated with the repair operation.",
+          "type": "string"
+        },
+        "message": {
+          "description": "The result message associated with the repair operation.",
+          "type": "string"
+        }
+      }
+    },
+    "DevBoxRepairOutcome": {
+      "type": "string",
+      "enum": [
+        "FixApplied",
+        "IssuesDetected",
+        "NoIssuesDetected"
```

Comment:
Unless existing enums are already PascalCase, enum values should be camelCase as well. But remain consistent. Same as other new enums in this PR like the kind model below.

Reply:
Thanks for flagging this. I just took a look through our enums and it looks like we use PascalCase everywhere (with the exception of environment parameters, which are aligned with standard JSON types and so use camelCase), so I'll leave these for consistency.